tests/ops/api/v1/endpoints/test_connection_template_endpoints.py

```diff
@@ -42,108 +50,194 @@ def test_get_connection_types_forbidden(
         assert resp.status_code == 403
 
     def test_get_connection_types(
-        self, api_client: TestClient, generate_auth_header, url
+        self,
+        api_client: TestClient,
+        generate_auth_header,
+        url,
+        saas_template_registry: ConnectorRegistry,
     ) -> None:
         auth_header = generate_auth_header(scopes=[CONNECTION_TYPE_READ])
         resp = api_client.get(url, headers=auth_header)
         data = resp.json()["items"]
         assert resp.status_code == 200
-        assert len(data) == 24
+        assert (
+            len(data)
+            == len(ConnectionType) + len(saas_template_registry.connector_types()) - 4
+        )  # there are 4 connection types that are not returned by the endpoint
```

Comment:
Which connections types are not returned?

Reply:
we filter out these 5 connection types [here](https://github.com/ethyca/fides/blob/unified-fides-2/src/fides/api/ops/api/v1/endpoints/connection_type_endpoints.py#L54-L58):

```
ConnectionType.saas,
ConnectionType.https,
ConnectionType.manual,
ConnectionType.email,
ConnectionType.manual_webhook,
```

but then re-add `manual_webhook` [here](https://github.com/ethyca/fides/blob/unified-fides-2/src/fides/api/ops/api/v1/endpoints/connection_type_endpoints.py#L98-L116). i'm not 100% sure why it works this way.

obviously this piece of the test is a bit less dynamic and less robust to changes moving forward. but it seems reasonable to me that the test would need to be updated when that code changes - it's a substantive functionality change, of a different variety than simply adding in a new saas connector.

Reply:
I agree, this is completely reasonable, it will reduce the amount of times we have to come visit these tests and that's all we need